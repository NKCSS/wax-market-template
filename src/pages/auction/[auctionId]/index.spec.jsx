import { render, screen } from '@testing-library/react'
import AuctionIdPage from './index.page'

describe('::AuctionIdPage', () => {
    it('renders as an error', async () => {
        // no asset given
        const { container } = render(<AuctionIdPage />)
        expect(screen.queryByTestId('error')).not.toBeNull()
        expect(container).toMatchSnapshot()
    })

    // describe('with asset', () => {
    //     /**
    //      * @type {import('../../../api/fetch').Asset}
    //      */
    //     let asset
    //     beforeEach(() => {
    //         asset = fixtures.asset()
    //     })

    //     it('renders', async () => {
    //         const { container } = render(<AuctionIdPage asset={asset} />)
    //         expect(screen.queryByTestId('error')).toBeNull()
    //         expect(container).toMatchSnapshot()
    //     })
    // })
})
