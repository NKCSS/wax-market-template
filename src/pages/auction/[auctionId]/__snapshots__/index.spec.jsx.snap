// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`::AuctionIdPage renders as an error 1`] = `
<div>
  <p
    data-testid="error"
  >
    An error occurred on client
  </p>
</div>
`;
